### DISCUSSION TITLE
Hubitat as Home Assistant radio ?

### DISCUSSION THREAD

**Discussion by luzik** [jason0x43/hacs-hubitat#111] [General]
I am new in home automation subject and just in begin of buying first hardware.

1. Am I understand correctly that this software allows me to use hubitat as reliable z-wave Zigbee Home Assistant hardware ?
2. Is it good idea to use it in that way, or better to use dedicated usb stick radio ?

I have to decide what hardware to buy and Hubitat Elevation hub C-7 (z-wave 700) sound like nice option instead of HUSBZB-1 (z-wave 500)

**Comment by jason0x43** [comment]
1. This software does allow you to use a Hubitat as Zigbee / Z-wave bridge (that was the main reason I created it). It allows you to control devices that are connected to Hubitat and respond to device events, but it does not give you direct control over the Zigbee or Z-wave hardware of the Hubitat.
2. I have not used a dedicated USB stick, only a Hubitat + this integration. On one hand, I believe that it's probably easier to manage devices through a Hubitat than through HA; Hubitat has a simpler UI and more mature software stack than HA, it has broad device support, and as a commercial product it is sell supported (it receives frequent updates, and you can contact support if you have problems). On the other hand, a Hubitat hub is a good deal more expensive than a USB stick, and with a USB stick you'd also have more control over the Zigbee and Z-wave radios, which can be useful to diagnose and resolve network issues.
